app/models/conversation.rb

```diff
@@ -33,6 +33,17 @@ def hiring_fee_eligible?
     developer_replied? && created_at <= 2.weeks.ago
   end
 
+  def latest_message_read_by_other_user?(other_user)
```

Comment:
I think this can be refactored to `#latest_message_read_by_other_recipient?` where we pass in the current user and use `#other_recipient) to get the other recipient.